src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -14,7 +14,7 @@ public static class Utilities
         /// <summary>
         /// Key words for Products containing ASM binaries
         /// </summary>
-        private static readonly List<string> ProductsWithASM = new List<string>() { "Revit", "Civil", "FormIt" };
+        private static readonly List<string> ProductsWithASM = new List<string>() { "Revit", "Civil", "Robot Structural Analysis", "FormIt" };
```

Comment:
ASM from RSA will be used if there is no Revit or Civil3D installed on the user's machine.

Reply:
I forgot if changing the order here affect the ASM paths found?

Reply:
Yes, it does. See: https://github.com/DynamoDS/Dynamo/blob/7a392ef25a532e0782caf57876aa43f0e4fc7ac2/src/Tools/DynamoShapeManager/Utilities.cs#L154

We first iterate over the different LibG versions and then over the supported products list and whichever matches first, comes out the winner. So it does matter if we play with the order. For example if Revit and RSA are both installed, and there is a match, ASM will be loaded from Revit and if we swap Revit with RSA, ASM will be loaded from RSA.

Reply:
Thanks, this is refreshing to me. Keeping the order of Revit first looks safe.